src/main/java/frc/robot/commands/SeekTargetWithLimelight.java

```diff
@@ -50,6 +52,9 @@ public SeekTargetWithLimelight(
   public void initialize() {
     this.oldPipelineNumber = limelight.getLimelightPipeline();
     limelight.setLimelightPipeline(seekingPipelineNumber);
+    if (DriverStation.getAlliance().equals(Alliance.Red)) {
```

Comment:
You'll need to do `DriverStation.getAlliance().get().equals(Alliance.Red)` because `DriverStation.getAlliance()` returns an [Optional](https://docs.oracle.com/en/java/javase/21/docs/api/java.base/java/util/Optional.html)

Reply:
Also add a blank line above here for spacing too